### DISCUSSION TITLE
Source query highlight at Provider data dropdown stopped working

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12132] [Help] [answered]
Source query highlight at Provider data dropdown stopped working. Budibase version. 2.11.33.

**Comment by ConorWebb96** [comment]
Hey @gmentzik,

Any chance you could provide some information on what you are talking about? (Screenshots as well) 1 liners really aren't helpful.

Thanks ahead of time.

**Comment by gmentzik** [comment]
![image](https://github.com/Budibase/budibase/assets/65217733/67ca43ce-9944-4b93-9900-c7b4ab7a5c9f)
Please check the image. At budibase 2.8 the selected query or table for provider source used to be highlighted (white background) as a solution that Data field cannot display long name. Until you add some kind of tooltip for these long name cases this is the only way to find which query is used. Another solution would be to display the Query at the window that opens when hitting cog. Now only Bindings and query appear.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @gmentzik,

I've added the the colour highlighted to show which one has been selected. So whenever this [pr](https://github.com/Budibase/budibase/pull/12196) gets merged in you'll be able to easily see which one is being used.

I hope this helps.
